File: 프로그래머스/unrated/181915. 글자 이어 붙여 문자열 만들기/글자 이어 붙여 문자열 만들기.js
function solution(my_string, index_list) {
    var answer = '';
   
    console.log(typeof(my_string));
    for(var i=0; i<index_list.length; i++){
       // answer += my_string[index_list[i] ] ;
         //문자열[index] 대괄호 인덱스를 활용하여 index의 문자를 가져올수있다?
       
        //또는 charAt() 이라는 함수를 사용할 수 있다.
        answer += my_string.charAt(index_list[i]);
       
    }
    return answer;
}

File: 프로그래머스/unrated/181915. 글자 이어 붙여 문자열 만들기/글자 이어 붙여 문자열 만들기.java
import java.util.*;

class Solution {
    public String solution(String my_string, int[] index_list) {
        String answer = "";
        
        String[] str = my_string.split("");
        System.out.println(Arrays.toString(str));//z, p, i, a
        //System.out.println(str.toString); 했더니 주소값?이 나왔었다.

        for(int i=0; i<index_list.length; i++){
            answer += str[index_list[i]];
        }
        return answer;
    	}
    }